package.json

```diff
@@ -6,6 +6,12 @@
     "private": true,
     "dependencies": {
         "@babel/standalone": "^7.21.4",
+        "@ckeditor/ckeditor5-alignment": "^40.1.0",
```

Comment:
related to my cross division dependency comment below, can you please make a note of these dependencies in confluence here: https://confluence.dimagi.com/display/saas/CommCareHQ+dependency+management#CommCareHQdependencymanagement-CommunicationAcrossDivisions

Thanks!

Reply:
I can do this - not sure Farid has edit access to confluence

Reply:
@biyeun I see in that page
>  the author is expected to receive sign-off from at least three Dependency Pod members before proceeding

Does what I see above reflect that consensus?

> [biyeun](https://github.com/biyeun) reviewed on behalf of https://github.com/orgs/dimagi/teams/dependency-watchers Dec 14, 2023

Or should we expect to see :heavy_check_mark: from 3 individual github accounts?

Reply:
raising to chat, brb

Reply:
I think that's enough for me (or one person for dependapod) to give a review and signoff, especially during people's offline schedules right now

Reply:
Thanks for handling!

Reply:
Since Biyeun is offline I can sign off from the dependency perspective.

Reply:
Which is to say, this looks good to me. I can also approve the PR just to be explicit.